src/main/java/org/opensearch/knn/index/memory/NativeMemoryEntryContext.java

```diff
@@ -55,13 +59,27 @@ public String getKey() {
      */
     public abstract Integer calculateSizeInKB();
 
+    /**
+     * Opens the graph file by opening the corresponding indexInput so
+     * that it is available for graph loading
+     */
+
+    public void openVectorIndex() {}
```

Comment:
Can we just call this open? The index aspect only really applies to IndexEntryContext. There are other allocation types this might not make sense for